src/reducers/widgets/index.js

```diff
@@ -0,0 +1,28 @@
+import { CLOCK_TICK } from '../../actions/widgets'
+import clockTicked from './clockReducer'
+
+import dummyClock from './dummy-data/dashboard-tools/clock'
+
+const updateWidget = ( state, action, type, updater ) => {
+  let widgetIndex = 0
+  while ( state[ widgetIndex ].type !== type ) {
+    widgetIndex++
+  }
+
+  return [
+    ...state.slice( 0, widgetIndex ),
+       updater(state[ widgetIndex ], action),
+    ...state.slice( widgetIndex + 1 )
+  ]
+}
+
+const widgetReducer = ( state = [dummyClock], action ) => {
```

Comment:
I think you should import and add `dummyText, dummyQRCode, 
dummyImage` to this array.  When a new person starts working on this project, they will see the range of widgets that have been made and will become excited by how awesome dragonboard is.